### DISCUSSION TITLE
增加支持传送bytes的接口

### DISCUSSION THREAD

**Discussion by Hieuzest** [howmanybots/onebot#81] [想法 / 建议]
用来传送Image, Voice, Video, File, etc.

**Comment by ExerciseBook** [comment]
好想法。

现阶段根据文档里的推荐写法，是可以使用 Base64 来暂时顶着。
https://github.com/howmanybots/onebot/blob/master/v11/specs/message/segment.md#%E5%9B%BE%E7%89%87

虽然感觉可能不太好看，

**Comment by Hieuzest** [comment]
当前方案里，base64对于大文件的效率很低，而转存临时文件有平台限制，对于框架和实现分属不同机器的场合也需要引入额外的文件传输协议。

**Comment by ExerciseBook** [comment]
@Hieuzest 我也觉得base64 可能在大文件里特别不划算

**Comment by ExerciseBook** [comment]
https://github.com/howmanybots/onebot/issues/47
关联话题
